packages/web/src/pages/purchases-and-sales/utils.ts

```diff
@@ -0,0 +1,5 @@
+import { PurchaseRow } from './types'
+
+export const isEmptyRow = (row?: PurchaseRow) => {
```

Comment:
super nit: Maybe `isEmptyPurchaseRow` to better hint/guard against auto-imports from other tables

Reply:
Oh good suggestion.